clicache/src/impl/PdxReaderWithTypeCollector.cpp

```diff
@@ -99,45 +99,45 @@ namespace Apache
           return retVal;
         }
 
-        System::UInt16 PdxReaderWithTypeCollector::ReadUInt16(String^ fieldName)
+        System::UInt16 PdxReaderWithTypeCollector::ReadUShort(String^ fieldName)
         {
-          checkType(fieldName, PdxFieldTypes::SHORT, "short");
-          m_newPdxType->AddFixedLengthTypeField(fieldName, "short", PdxFieldTypes::SHORT, native::PdxTypes::SHORT_SIZE);
+          checkType(fieldName, PdxFieldTypes::SHORT, "ushort");
+          m_newPdxType->AddFixedLengthTypeField(fieldName, "ushort", PdxFieldTypes::SHORT, native::PdxTypes::SHORT_SIZE);
           int position = m_pdxType->GetFieldPosition(fieldName, m_offsetsBuffer, m_offsetSize, m_serializedLength);
           if (position != -1)
           {
             m_dataInput->AdvanceCursorPdx(position);
-            UInt16 retVal = PdxLocalReader::ReadUInt16(fieldName);
+            UInt16 retVal = PdxLocalReader::ReadUShort(fieldName);
             m_dataInput->RewindCursorPdx(position);
             return retVal;
           }
           return 0;
         }
 
-        System::UInt32 PdxReaderWithTypeCollector::ReadUInt32(String^ fieldName)
+        System::UInt32 PdxReaderWithTypeCollector::ReadUInt(String^ fieldName)
```

Comment:
Again, this renaming feels gratuitous.  Also I really hate renaming UInt64 -> ULong.  It feels like we're losing important self-documentation.

Reply:
Same as above. Microsoft uses the C# keywords in their API docs.